src/App.js

```diff
@@ -1,9 +1,5 @@
 import "./App.css";
-import { Cart } from "./frontend/pages/cart/cart";
-import { Home } from "./frontend/pages/home/Home";
-import { Signup } from "./frontend/pages/signup/signup";
-import { Wishlist } from "./frontend/pages/wishlist/Wishlist";
-import { Login } from "./frontend/pages/login/login";
```

Comment:
import from the index of seperate folder, dont hardcode it

Reply:
ok, will do it.